### DISCUSSION TITLE
Is Streaming In Higher Resolution Possible(Python preview demo is low res)?

### DISCUSSION THREAD

**Discussion by AustinFoss** [gopro/OpenGoPro#133] [General]
I have a project in mind that involves streaming from a GoPro and I would like to use open-gopro for it.

My question comes from after getting the preview window to show up in VLC I can't seem to get it to increase in the resolution at all.

I know we can stream in 1080 from the official app so it's possible but there doesn't seem to be anything in the API to enable this.

Does the official app transfer the higher res stream via a different method aside from the UDP stream at Port 8554?

Thanks!

**Comment by KonradIT** [comment]
What resolution where you trying to hit?

So the official app uses the UDP packets over wifi for showing a preview, and you can change the resolution up to 1080p using the webcam mode (via usb, documented in this repo) or using `http://10.5.5.9/gp/gpControl/setting/64/12` over WiFi (for Hero9 and 10).

**Reply by AustinFoss** [reply]
Oh darn, I didn't realize this wasn't usable on a hero 8. 

Still I know there must be command I can use to get the quality up because the app can send the signal using the app to get a 1080p stream going. Is there any way to find out what that get request is that's being sent from the official app to a Hero 8?

**Reply by KonradIT** [reply]
Try it with the 8. I might be mistaken.
